.travis.yml

```diff
@@ -29,8 +29,12 @@ addons:
       - xclip
 
 install:
+  - cd $HOME
+  - mkdir -p $HOME/gopath/src/github.com/limetext
+  - mv ${TRAVIS_BUILD_DIR} $HOME/gopath/src/github.com/limetext/backend
+  - cd $HOME/gopath/src/github.com/limetext/backend
+  - export TRAVIS_BUILD_DIR="$HOME/gopath/src/github.com/limetext/backend"
```

Comment:
Sorry, my bad. I was testing on a fork and I forgot to remove this before doing the pull request. I'll fix it later today.